src/libs/ajax/leonardo/Runtimes.ts

```diff
@@ -256,3 +260,27 @@ export const Runtimes = (signal) => {
     },
   };
 };
+
+// const MAX_RETRIES = 5;
+// const TIMEOUT_DURATION = 10000;
+
+// function makeRequestRetry(request, retriesLeft) {
+//   return Promise.race([
+//     request(),
+//     new Promise((_, reject) => setTimeout(() => reject(new Error('Request timed out')), TIMEOUT_DURATION)),
+//   ])
+//     .then((response) => {
+//       if (response.ok) {
+//         return response.json();
+//       }
+//       throw new Error('Network response was not ok.');
+//     })
```

Comment:
any reason why this was commented and not deleted?

Reply:
I wanted a bigger diff. 🤷